### PR TITLE
[INFRA] Bump Node version from 14 to 16

### PR DESCRIPTION
**Pull request by csouchet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This site is created with Gatsby.
 
 1.  **Start developing.**
 
-    Mininal node version: `14.15.0` (for Gatsby v4)
+    Mininal node version: `16.13.0` (for Gatsby v4)
```
Comment (tbouffard) [inline README.md:29]: Gasby v4 requires 14.5.0+ according to https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v3-to-v4/#minimal-nodejs-version-14150
The page doesn't mention any minima v16 version

**Review by tbouffard — COMMENTED** [review]
❓ Could we add a `.nvmrc` file in the repository to help using the right version? https://github.com/nvm-sh/nvm#nvmrc

File: README.md (hunk 1 above)
Comment (csouchet) [reply README.md:29]: The minimal version we specify here is not only for the whole project ?
Because, we choose to build it only from v16.

Comment (csouchet) [reply README.md:29]:
```suggestion
    Mininal node version: `16.x`
```

**Review by tbouffard — APPROVED** [review]

**Pull request merged**